src/test/regress/expected/multi_colocation_utils.out

```diff
@@ -716,93 +716,129 @@ ORDER BY
- table1_group_none_1              | 1300055 | t            |    57637 | 0             | 2147483647
- table2_group_none_1              | 1300056 | t            |    57638 | -2147483648   | -1
- table2_group_none_1              | 1300056 | t            |    57637 | -2147483648   | -1
- table2_group_none_1              | 1300057 | t            |    57637 | 0             | 2147483647
- table2_group_none_1              | 1300057 | t            |    57638 | 0             | 2147483647
- table1_group_none_2              | 1300058 | t            |    57637 | -2147483648   | -1
- table1_group_none_2              | 1300058 | t            |    57638 | -2147483648   | -1
- table1_group_none_2              | 1300059 | t            |    57638 | 0             | 2147483647
- table1_group_none_2              | 1300059 | t            |    57637 | 0             | 2147483647
- table4_groupe                    | 1300060 | t            |    57637 | -2147483648   | -1
- table4_groupe                    | 1300060 | t            |    57638 | -2147483648   | -1
- table4_groupe                    | 1300061 | t            |    57638 | 0             | 2147483647
- table4_groupe                    | 1300061 | t            |    57637 | 0             | 2147483647
- table1_group_none_3              | 1300062 | t            |    57637 | -2147483648   | -715827884
- table1_group_none_3              | 1300062 | t            |    57638 | -2147483648   | -715827884
- table1_group_none_3              | 1300063 | t            |    57638 | -715827883    | 715827881
- table1_group_none_3              | 1300063 | t            |    57637 | -715827883    | 715827881
- table1_group_none_3              | 1300064 | t            |    57637 | 715827882     | 2147483647
- table1_group_none_3              | 1300064 | t            |    57638 | 715827882     | 2147483647
- table1_group_default             | 1300065 | t            |    57637 | -2147483648   | -715827884
- table1_group_default             | 1300065 | t            |    57638 | -2147483648   | -715827884
- table1_group_default             | 1300066 | t            |    57638 | -715827883    | 715827881
- table1_group_default             | 1300066 | t            |    57637 | -715827883    | 715827881
- table1_group_default             | 1300067 | t            |    57637 | 715827882     | 2147483647
- table1_group_default             | 1300067 | t            |    57638 | 715827882     | 2147483647
- table1_groupf                    | 1300068 | t            |    57637 |               | 
- table1_groupf                    | 1300068 | t            |    57638 |               | 
- table2_groupf                    | 1300069 | t            |    57637 |               | 
- table2_groupf                    | 1300069 | t            |    57638 |               | 
-(84 rows)
+                     logicalrelid                      | shardid | shardstorage | nodeport | shardminvalue | shardmaxvalue 
```

Comment:
Why didn't we set GUCs here?

Reply:
Missed this one. I did skip a few because it would be too inconvenient to set the GUC (e.g. they reconnect very often). Those should remain sequential for the moment.